protocol/consensus.go

```diff
@@ -192,11 +192,22 @@ const ConsensusV35 = ConsensusVersion(
 	"https://github.com/algorandfoundation/specs/tree/433d8e9a7274b6fca703d91213e05c7e6a589e69",
 )
 
-// ConsensusV36 adds box storage
+// ConsensusV36 adds box storage in TEAL v8
 const ConsensusV36 = ConsensusVersion(
 	"https://github.com/algorandfoundation/specs/tree/44fa607d6051730f5264526bf3c108d51f0eadb6",
 )
 
+// ConsensusV37 enables AVM resources sharing and pre-check ECDSA curve
```

Comment:
Does V37 enable sharing?

Reply:
no, it does not. Looks like copy-paste from the previous version where it did. will fix